en/editor/publish/build-open-data-context.md

```diff
@@ -0,0 +1,113 @@
+# Building Open Data Context Project
+
+## Introduction to Open Data Context
+
+At present, **WeChat**, **Baidu** and **ByteDance** are small game platforms that add the concept of **Open Data Context**, which is a separate game execution environment, in order to protect their social relationship chain data. The resources, engines, and applications in the Open Data Context are completely isolated from the main game, and only in the Open Data Context can developers access the relationship chain data through the open interface provided by the platform to implement some features such as ranked version.
```

Comment:
in order
-->
in-order

Reply:
I wonder why we should use in-order here, Is there a misunderstanding here?

Reply:
no, don't, my mistake.